### PR TITLE
Ingestion  & lambda best practices

### PR DESCRIPTION
**Pull request by thecraftman**
fixes for: https://github.com/axiomhq/docs/pull/105

### CODE REVIEW

**Comment by manototh** [comment]
@thecraftman Thanks a lot for this! The additions to the Send data overview page are spot on!

I'm not sure about the changes to the Lambda page. I've read through the comments in the closed PR (#105) and it seems to me that we should clearly explain what the best course of action is depending on the user's situation. The [AWS overview page](https://axiom-ingestion-lambda-best-practices.mintlify.app/send-data/aws-overview#choose-the-best-method-to-send-data) does this and we link to it at the top of the Lambda page. The changes you propose to this page outline the same options but don't tell the user about the pros and cons. I think this isn't necessary. Or maybe I've missed the original motivation? Thanks in advance for clarifying this.

**Comment by thecraftman** [comment]
> @thecraftman Thanks a lot for this! The additions to the Send data overview page are spot on!
> 
> I'm not sure about the changes to the Lambda page. I've read through the comments in the closed PR (#105) and it seems to me that we should clearly explain what the best course of action is depending on the user's situation. The [AWS overview page](https://axiom-ingestion-lambda-best-practices.mintlify.app/send-data/aws-overview#choose-the-best-method-to-send-data) does this and we link to it at the top of the Lambda page. The changes you propose to this page outline the same options but don't tell the user about the pros and cons. I think this isn't necessary. Or maybe I've missed the original motivation? Thanks in advance for clarifying this.

@tothmano missed that as I didn't check the page just discovered we have an AWS overview page.  changes now updated

**Review by manototh — APPROVED** [review]

File: send-data/ingest.mdx [hunk 1]
```diff
@@ -292,3 +292,10 @@ All events stored in Axiom must have a `_time` timestamp field. If the data you
...
+
+When sending data into Axiom, follow these best practices to optimize performance and reliability:
+- **Batch events:** Use a [log forwarder](https://axiom.co/docs/send-data/ingest), [collector](https://axiom.co/docs/send-data/ingest#data-shippers), or [Axiom‘s official SDKs](https://axiom.co/docs/send-data/ingest#client-libraries) to group multiple events into a single request before sending them to Axiom. This reduces the number of API calls and improves overall throughput. Avoid implementing batching within your app itself as this introduces additional complexity and requires careful management of buffers and error handling.
+- **Use compression:** Enable gzip, zstd compression for your requests to reduce bandwidth usage and potentially improve response time.
+- **Handle rate limiting and errors:** Use [Axiom‘s official libraries and SDKs](https://axiom.co/docs/send-data/ingest#client-libraries) which automatically implement best practices for handling rate limiting and errors. For advanced use cases or custom implementations, consider adding a fallback mechanism to store events locally or in cold storage if ingestion consistently fails after retries.
```
Comment (manototh) [inline send-data/ingest.mdx]:
```suggestion
When sending data into Axiom, follow these best practices to optimize performance and reliability:
- **Batch events:** Use a log forwarder, [collector](#data-shippers), or [Axiom‘s official SDKs](#client-libraries) to group multiple events into a single request before sending them to Axiom. This reduces the number of API calls and improves overall throughput. Avoid implementing batching within your app itself as this introduces additional complexity and requires careful management of buffers and error handling.
- **Use compression:** Enable gzip, zstd compression for your requests to reduce bandwidth usage and potentially improve response time.
- **Handle rate limiting and errors:** Use [Axiom‘s official libraries and SDKs](#client-libraries) which automatically implement best practices for handling rate limiting and errors. For advanced use cases or custom implementations, consider adding a fallback mechanism to store events locally or in cold storage if ingestion consistently fails after retries.
```

**Comment by manototh** [comment]
@thecraftman Great, thanks, merge at will!

**Pull request merged**